buildlib/RDMA_EnableCStd.cmake

```diff
@@ -85,3 +85,43 @@ int main(int argc, char *argv[])
 
   set(${TO_VAR} "${HAVE_WORKING_STRICT_ALIASING}" PARENT_SCOPE)
 endfunction()
+
+function(RDMA_Check_SSE TO_VAR)
+  CHECK_C_SOURCE_COMPILES("
+    #include <xmmintrin.h>
```

Comment:
This all looks fine now,

But I still think this block needs to be enclosed in #if defined(__i386__) so it doesn't fail on other architectures. The 'SEE extensions do NOT work' should only be printed if it is a problem, and it isn't a problem on arm, etc.